server/src-lib/Hasura/Server/Utils.hs

```diff
@@ -212,7 +212,7 @@ httpExceptToJSON e = case e of
 -- ignore the following request headers from the client
 commonClientHeadersIgnored :: (IsString a) => [a]
 commonClientHeadersIgnored =
-  [ "Content-Length", "Content-MD5", "User-Agent", "Host"
+  [ "Content-Length", "Content-MD5", "Host"
```

Comment:
Why is this removed?

Reply:
https://github.com/hasura/graphql-engine/issues/2572

Reply:
I am not sure if this was the best way. Review requested.

Reply:
I don't think we can forward the original `user-agent` as is given that we are not a proxy (currently we kind of are but we won't be in the future when you have remote joins). Maybe we can add headers like `hge-client-user-agent` and `hge-client-ip`?

Reply:
Okay, I am just going to revert this commit. And take it in a separate PR